network/quic/dialer.go

```diff
@@ -0,0 +1,37 @@
+package quic
+
+import (
+	"crypto/tls"
+	"time"
+
+	h "github.com/ConsenSys/handel"
+	quic "github.com/lucas-clemente/quic-go"
+)
+
+//Dialer is an interface responsible for creating session between two peers
+type Dialer interface {
```

Comment:
1. Is this intended to be implemented externally ? If not, then we should not put it as publicly exposed.
2. Could you write just a small sentence describing which implementation of that interface exists at that time ?